actnow/petitions/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from actnow.accounts.models import ActNowUser
+from actnow.db.models import TimestampedModelMixin
+
+
+class Petition(TimestampedModelMixin):
+    title = models.CharField(
+        _("title"),
+        max_length=255,
+    )
+    description = models.TextField(_("details of petition"), blank=True)
+    owner = models.ForeignKey(
+        ActNowUser,
+        on_delete=models.CASCADE,
+    )
+    recipients = models.CharField(
+        _("recipients"),
+        max_length=255,
+    )
+    problem_to_address = models.TextField(
+        _("problem to be addressed"),
+    )
```

Comment:
```suggestion
    problem_statement = models.TextField(
        _("problem to be addressed"),
    )
```
?
Lets also have some max_length limit here... 512, 1024, etc.